### PR TITLE
Add web3js-quorum content

### LINKED ISSUE #114: quorum.js and web3js-eea superseded by web3js-quorum
quorum.js and web3js-eea deprecation is officially announced for December 31st  2021.  From now on we will be supporting [web3js-quorum](https://github.com/ConsenSys/web3js-quorum) which is basically a merge and refacto of the two former libraries (all commits have been kept) that is compatible with both GoQuorum and Besu.

Some guides and tutorials may need to be updated.

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Removed deprecated quorum.js content and replaced with web3js-quorum.

## Fixed Issue(s)
Fixes #114 

## Preview

- https://pegasys-docgoquorum--129.com.readthedocs.build/en/129/HowTo/Interact/Client-Libraries/web3js-quorum/
- https://pegasys-docgoquorum--129.com.readthedocs.build/en/129/Reference/web3js-quorum/

### CODE REVIEW

File: docs/Reference/quorum.js/RawTransactionManager.md [hunk 1]
```diff
@@ -1,213 +0,0 @@
----
```
Comment (bgravenorst) [inline docs/Reference/quorum.js/RawTransactionManager.md:1]: The web3js-quorum equivalent of this (including method descriptions) can be found in the web3js-quorum documentation.

**Review by rolandtyler — APPROVED** [review]
LGTM

File: docs/HowTo/Interact/Client-Libraries/web3js-quorum.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+## Initialize the web3js-quorum client
+
+Initialize your client where:
+
+* `<GoQuorum JSON-RPC HTTP endpoint>` is the JSON-RPC HTTP endpoint of your GoQuorum node.
+* `<Enclave IPC Path>` is the enclave IPC Unix socket path
+* `<Enclave Private URL>` is the enclave HTTP endpoint
+* `<enclave key file path>` is the enclave public key file path, see [Tessera keys doc](https://docs.tessera.consensys.net/en/stable/HowTo/Generate-Keys/File-Stored-Keys/#store-keys-in-files)
+* `<enclave TLS cert file path>` is the enclave TLS client certificate file path, see [Tessera TLS doc](https://docs.tessera.consensys.net/en/stable/HowTo/Configure/TLS/)
+* `<enclave TLS CA cert file path>` is the enclave TLS certification authority (CA) certificate file path, see [Tessera TLS doc](https://docs.tessera.consensys.net/en/stable/HowTo/Configure/TLS/)
```
Comment (alexandratran) [inline docs/HowTo/Interact/Client-Libraries/web3js-quorum.md:44]:
```suggestion
* `<Enclave IPC Path>` is the enclave IPC Unix socket path.
* `<Enclave Private URL>` is the enclave HTTP endpoint.
* `<enclave key file path>` is the enclave public key file path (see [Tessera keys documentation](https://docs.tessera.consensys.net/en/stable/HowTo/Generate-Keys/File-Stored-Keys/#store-keys-in-files)).
* `<enclave TLS cert file path>` is the enclave TLS client certificate file path (see [Tessera TLS documentation](https://docs.tessera.consensys.net/en/stable/HowTo/Configure/TLS/)).
* `<enclave TLS CA cert file path>` is the enclave TLS certification authority (CA) certificate file path (see [Tessera TLS documentation](https://docs.tessera.consensys.net/en/stable/HowTo/Configure/TLS/)).
```

File: docs/Reference/web3js-quorum.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+Description: Web3js-quorum library
+---
+
+# Web3js-quorum
+
+Web3js-quorum is an Ethereum JavaScript library extending [web3.js] that adds supports for GoQuorum and Hyperledger Besu
```
Comment (alexandratran) [inline docs/Reference/web3js-quorum.md:7]: Need consistency on capitalization of "web3js-quorum" throughout this page and with the client library page.

**Pull request merged**